src/modules/vtol_att_control/standard.cpp

```diff
@@ -94,6 +96,12 @@ Standard::parameters_update()
 
 	_airspeed_trans_blend_margin = _params_standard.airspeed_trans - _params_standard.airspeed_blend;
 
+	/* timeout for transition to fw mode */
+	param_get(_params_handles_standard.front_trans_timeout, &v);
+	_params_standard.front_trans_timeout= math::constrain(v, 0.0f, 20.0f);
```

Comment:
I don't think we should constrain it to a max. The max value in the parameter definition is fine still (maybe increase to 30), only used for the GUI to give the user an idea.